soc/xtensa/esp32/dts_fixup.h

```diff
@@ -8,4 +8,5 @@
 
 #define DT_CPU_CLOCK_FREQUENCY DT_CADENCE_TENSILICA_XTENSA_LX6_0_CLOCK_FREQUENCY
 #define CONFIG_ENTROPY_NAME DT_INST_0_ESPRESSIF_ESP32_TRNG_LABEL
+#define CONFIG_WDT_0_NAME DT_INST_0_ESPRESSIF_ESP32_WATCHDOG_LABEL
```

Comment:
`CONFIG_WDT_0_NAME` is only defined if `HAS_DTS_WDT` is not enabled. So there is no need to define this here.

Reply:
i think it should be replaced by `DT_WDT_0_NAME`, but without both tests don't compile

Reply:
Other boards simply do something like `#define DT_WDT_0_NAME DT_INST_0_ESPRESSIF_ESP32_WATCHDOG_LABEL` in their `dts_fixup.h`